test_utils/scripts/update_docs.sh

```diff
@@ -23,8 +23,9 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 # Function to build the docs.
 function build_docs {
-    rm -rf docs/_build/
-    sphinx-build -W -b html -d docs/_build/doctrees docs/ docs/_build/html/
+    #rm -rf docs/_build/
+    #sphinx-build -W -b html -d docs/_build/doctrees docs/ docs/_build/html/
```

Comment:
1. Why did you remove the `rm -rf`?
1. Please delete rather than comment out these lines.
1. Please don't remove the `-W` flag (@lukesneeringer not sure what came of my previous comment about this)